go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/srl-labs/clabernetes
 
-go 1.22
+go 1.22.1
```

Comment:
I cant remember what we decided on this --adding the .1 fixed the toolchain issue for like local build/dev stuff I think, but also feels silly to pin minor versions I guess. refresh my memory because vacation so now I remember nothing :D

Reply:
yeah I failed to find a way to make vscode not going crazy without pinning to the last dot

but also pinning to 1.22.1 is the same as 1.22; it just says that any 1.22.x version >= 1.22.1 is fine, so I failed to see a major difference. Even more granularity if there will ever be a need to say that a certain patch release must be used by the consumers.

Reply:
works for me. I dont think it should matter much anyway since we build the only images that ppl would use anyway!